dev/io.openliberty.mcp.internal/src/io/openliberty/mcp/internal/McpServlet.java

```diff
@@ -81,19 +81,19 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         McpTransport transport = new McpTransport(req, resp, jsonb);
         String accept = req.getHeader("Accept");
         // Return 405, with SSE-specific message if "text/event-stream" is requested.
-        if (accept != null && HeaderValidation.acceptContains(accept, "text/event-stream")) {
-            HttpResponseException e = new HttpResponseException(
-                                                                HttpServletResponse.SC_METHOD_NOT_ALLOWED,
-                                                                "GET not supported yet. SSE not implemented.")
-                                                                                                              .withHeader("Allow", "POST");
-            transport.sendHttpException(e);
-        } else {
-            HttpResponseException e = new HttpResponseException(
-                                                                HttpServletResponse.SC_METHOD_NOT_ALLOWED,
-                                                                "GET method not allowed.")
-                                                                                          .withHeader("Allow", "POST");
-            transport.sendHttpException(e);
-        }
+//        if (accept != null && HeaderValidation.acceptContains(accept, "text/event-stream")) {
+//            String excpetionMesaage = Tr.formatMessage(tc, "CWMCM0008I.sse.unimplemented");
+//            HttpResponseException e = new HttpResponseException(
+//                                                                HttpServletResponse.SC_METHOD_NOT_ALLOWED,
+//                                                                excpetionMesaage).withHeader("Allow", "POST");
+//            transport.sendHttpException(e);
+//        }
```

Comment:
Just delete this code rather than commenting it out.

We can always get it back from the history if we decide we need it.